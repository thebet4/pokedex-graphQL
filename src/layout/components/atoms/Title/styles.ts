import { colors } from "@config/theme";
import styled from "styled-components/native";

export const TitleText = styled.Text`
  color: ${colors.purple[200]}
  font-family: "Inter_700Bold";
  font-size: 20px;
  line-height: 24px;
  text-align: left;
  letter-spacing: 0px;
`;
