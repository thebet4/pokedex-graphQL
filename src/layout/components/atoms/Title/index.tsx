import React, { ReactNode } from "react";
import { TextStyle } from "react-native";
import { TitleText } from "./styles";

interface ITitleProps {
  children: ReactNode;
  props?: TextStyle;
}

const Title = ({ children, props }: ITitleProps) => {
  return <TitleText {...props}>{children}</TitleText>;
};

export default Title;
