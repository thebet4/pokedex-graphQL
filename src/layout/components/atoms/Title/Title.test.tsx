import * as React from "react";
import renderer from "react-test-renderer";

import Title from "@layout/components/atoms/Title";

it(`renders <Title /> correctly`, () => {
  const tree = renderer.create(<Title>Pokemon</Title>).toJSON();

  expect(tree).toMatchSnapshot();
});
