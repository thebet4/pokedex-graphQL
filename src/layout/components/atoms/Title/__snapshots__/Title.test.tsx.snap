// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Title /> correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#21386E",
        "fontFamily": "Inter_700Bold",
        "fontSize": 20,
        "letterSpacing": 0,
        "lineHeight": 24,
        "textAlign": "left",
      },
    ]
  }
>
  Pokemon
</Text>
`;
